docs/sdks/server-sdks/ruby.md

```diff
@@ -65,7 +109,17 @@ config = EppoClient::Config.new(
   '<YOUR_API_KEY>',
   assignment_logger: CustomAssignmentLogger.new
 )
-client = EppoClient::init(config)
+EppoClient::init(config)
+
+# Allow the configuration to be loaded from the CDN.
+sleep(1)
```

Comment:
🤨 no awaits or callbacks in ruby? or can we make a synchronous version?

Don't love seeing `sleep()` in our example 😞

Reply:
agreed, ruby is single threaded by default though and doesnt have async primitives like async/await. we could have a callback instead, but that would require changing the interface